### PR TITLE
Add package: Gnuboy

### PR DESCRIPTION
**Pull request by AnGelXoG**
Gnuboy emulator

### CODE REVIEW

**Comment by AnGelXoG** [comment]
finish code... xd

File: packages/gnuboy/makefile.patch [hunk 1]
```diff
@@ -0,0 +1,5 @@
+--- makefile    2021-11-11 23:56:47.461415526 -0600
++++ makefile.mod      2021-11-11 23:58:59.361415432 -0600
+@@ -1,25 +0,0 @@
+.PHONY: linux
+.PHONY: clean
```
Comment (xeffyr) [inline packages/gnuboy/makefile.patch:5]: This doesn't seem to be a valid patch file. Strongly recommending to create it with `diff` utility, especially if you don't know format internals.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Review by xeffyr — CHANGES_REQUESTED** [review]
Properly create the patch file.

1. Take the original sources and extract them
2. Copy: `cp -a sources.orig sources.mod`
3. `cd sources.mod`
4. Make changes
5. `cd ..`
6. Create patch file: `diff -uNr sources.orig sources.mod > patchfile.patch`

**Comment by xeffyr** [comment]
And still after a brief walkthrough about how to create patch, I don't see any progress and same mistakes as were before...
> Applying patch: makefile.patch
patch: **** unexpected end of file in patch

**Pull request closed without merging**